docs/how-to-guides/upgrade-bootstrap-kubelet.md

```diff
@@ -0,0 +1,44 @@
+# How to upgrade bootstrap kubelet
+
+## Introduction
+
+Bootstrap kubelet is the kubelet running on the node as a systemd service. Unlike most components on the cluster, lokoctl cannot update it. So there are some manual steps involved in updating the bootstrap kubelet.
+
+## Steps
+
+Apply the following steps to all the nodes one at a time.
+
+### Drain the node
+
```

Comment:
I'd add some explanation what does it do and why is it needed. If someone runs workload using local storage, it will be disturbed.

Reply:
Added an explanation.